requirements/base.txt

```diff
@@ -6,7 +6,7 @@
 git+https://github.com/edx/XBlock.git@9c634481dfc85a17dcb3351ca232d7098a38e10e#egg=XBlock
 
 # edx-submissions
-git+https://github.com/edx/edx-submissions.git@14aeaa9e30f9a408b34ffaf6d78409dedaad015a#egg=edx-submissions==0.1.0
+git+https://github.com/edx/edx-submissions.git@3d3ba2a4ba2d64f37dd1d908c8a1d78c1ac95523#egg=edx-submissions==0.1.1
```

Comment:
confirmed to be upgradeable via `make install` on an otherwise-already-set-up devstack.